src/test/java/io/supertokens/test/ConfigTest2_6.java

```diff
@@ -220,6 +220,11 @@ private static void checkConfigValues(CoreConfig config, TestingProcess process,
         assertFalse("Config access token blacklisting did not match default", config.getAccessTokenBlacklisting());
         assertEquals("Config refresh token validity did not match default", config.getRefreshTokenValidity(),
                 60 * 2400 * 60 * (long) 1000);
+        // TODO: Is this correct?
+        assertEquals(5, config.getTotpMaxAttempts()); // 5
+        assertEquals(900, config.getTotpRateLimitCooldownTime()); // 15 minutes
+        assertEquals(5, config.getTotpInvalidCodeExpiryTime()); // 30 minutes
+
```

Comment:
You need to also add tests where the user misconfigures the values and it throws an exception.

Reply:
Done.